CHANGELOG.md

```diff
@@ -154,6 +154,8 @@ Please follow the instructions before upgrading Cumulus
     The property `sf_event_sqs_to_db_records_types` has been added to
     `workflow_template.json` under the `cumulus_meta` field to control which record
     types should be written to the database during different workflow execution statuses.
+    Currently, both "execution" and "pdr" must be written to the database, so the
+    record type list must include both.
```

Comment:
I know we talked about this and I think these changes are good but I do wonder if it makes more sense, now that we have required input, to make it a list of records/status that are skipped rather than written. e.g. `sf_event_sqs_to_db_skip_record_writes`.

I don't think it's a deal-breaker either way as we're throwing an appropriate error but it could potentially be more confusing for users this way if they have options they really can't modify.

Reply:
If we think there's a case in the future to modify this logic to make it possible to remove other records, that makes more sense.